### ISSUE TITLE
Enchanter should have its own separated skin

### ISSUE THREAD

**Issue by warning-djkajslh** [ldtteam/minecolonies-features#427]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version:
- MineColonies Version:
- Related Mods and their Versions:

## Is your feature request related to a problem?

I don't like how University skin is being applied to multiple professions

## Enhancement or Change description

Enchanter should have a unique skin

### Notes or related things

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Moobien** [comment]
you are welcome to make a skin for it and submit a PR

**Comment by ravenbuilder934** [comment]
Barring that, do you have any suggestions of what it would look like?

**Comment by warning-djkajslh** [comment]
yes I would imagine they would be more wizardly, wearing magical clothing, a magician, not some common book researcher

**Comment by ravenbuilder934** [comment]
okay, well, if you want to create the 16 textures (including variations), you can. Our current texturers have a lot of work to do

**Comment by warning-djkajslh** [comment]
Nice but I am not in any hurry for the textures, so I will leave it to the crew :P
